app/views/layouts/application.html.erb

```diff
@@ -87,16 +85,16 @@
 
     <%= govuk_header(
       classes: govuk_header_classes(current_user),
-      service_url: current_user.nil? ? root_path : logs_path,
+      homepage_url: root_path,
```

Comment:
nice!

Reply:
once we've released the homepage this will go there rather than to the logs page, which is probably a good thing!